src/pharmpy/modeling/expressions.py

```diff
@@ -857,29 +854,52 @@ def has_random_effect(model: Model, parameter: str, level: str = 'all') -> bool:
     See also
     --------
     get_individual_parameters
+    get_rv_parameters
 
     """
 
     rvs = _rvs(model, level)
     return _depends_on_any_of(model, parameter, rvs)
 
 
-def get_rv_parameter(model: Model, rv: str) -> str:
-    # NOTE This was just copied here and is used elsewhere but should be made
-    # more general
-    sset = model.statements
+def get_rv_parameters(model: Model, rv: str) -> List[str]:
+    """Retrieves parameters in :class:`pharmpy.model` given a random variable.
 
-    s = _find_assignment(sset, sympy.Symbol(rv))
+    Parameters
+    ----------
+    model : Model
+        Pharmpy model to retrieve parameters from
+    rv : str
+        Name of random variable to retrieve
 
-    if s is None:
-        raise ValueError(f'Could not find an assignment to {rv}')
+    Return
+    ------
+    Parameters
+        The parameters of the random variable for the given model
```

Comment:
Return type is `list[str]`. This is also incorrect in the doc string of already present function. I think the idea is that since these are user-facing functions in R they should return reticulatable types.

Reply:
Agreed, have updated